packages/components/pie-data-table/src/data-table.scss

```diff
@@ -1,7 +1,72 @@
 @use '@justeattakeaway/pie-css/scss' as p;
 
 :host {
-    // Note: For consistency sake, the recommended display should be either
-    // "block" or "inline-block", otherwise it can be ommited for floating elements
     display: block;
 }
+
+.c-data-table {
+  --data-table-background-color: var(--dt-color-container-default);
+  --data-table-background-hover-or-active: var(--dt-color-container-subtle);
```

Comment:
do you know if in design we are using the container subtle or a combination of a color tokens and hover-1 or active-01? if the latter i think we should use them instead. there are some references in the codebase on how it's used. 

also it is expected that we show hover active styles when the row isn't clickable 👀